openml/_api_calls.py

```diff
@@ -1,22 +1,17 @@
-import io
-import os
+import time
 import requests
 import warnings
 
-import arff
 import xmltodict
 
 from . import config
 from .exceptions import (OpenMLServerError, OpenMLServerException,
                          OpenMLServerNoResult)
 
 
-def _perform_api_call(call, data=None, file_elements=None,
-                      add_authentication=True):
```

Comment:
don't you need the authentication flag? GET requests don't require authentication, POST/DELETE requests do.

Reply:
That is handled internally and does not need to be an argument to the openml-api call API.